tests/drone/wait-for-phoenix.sh

```diff
@@ -0,0 +1,7 @@
+curl -s phoenix:8300 > /dev/null
```

Comment:
we dont need this file .... I'm using a built-in script now ...

Reply:
removed it - your right. I missed that in rebase